content/en/dashboards/guide/context-links.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Context Links
+kind: guide
+further_reading:
+   - link: '/dashboards/widgets'
+     tag: 'Documentation'
+     text: 'List of Dashboard Widgets'
+
+---
+
+## Overview
+
+Your dashboards are fundamental in troubleshooting the root cause of an issue or determining an assertion to a critical question. Dashboard widgets can appear in your [notebooks][1], attached to a [monitor notification][2], or serve as a [screenboard][3]. 
+
+Use Datadog dashboards to perform an initial assessment of your context-relevant data sources. In third-party applications, you may also need to access external data or perform follow-up actions. Context links also allow you to access and trigger third-party applications through webhooks as well as override default links. 
+
+{{< img src="dashboards/guide/context_links/overview.mp4" alt="Context Link Demo" video="true" style="width:80%;">}}
+
+Dashboard widgets contain default links to relevant Datadog pages with information about the scope including the timerange, underlying queries, and the series on a graph. [Generic widgets][4] contain customizable links, allowing you to point to third-party applications and override the default links to customize your landing page. 
+
+This guide introduces context links and provides examples of how to create external links and override default links.
+
+## Add a context link
+
+By default, context links appear in the dropdown menu when you click on or in a widget in your dashboards and quick graphs. You can edit Out-of-the-box context links in the **Edit** tab under **Edit** > **Graph your data**. 
+
+To create a context link, click **+ Add a Context Link**.
+
+1. Add a description in the **Label** field. 
+2. Enter the URL to your external application in the **URL** field.
+3. Add key-value pairs by clicking **+ Add URL Parameter**.
+4. Click **Save**.
+
```

Comment:
```suggestion

### Template variables

Template variables that are available for context links include a widget filter, widget time range, dashboard template variables (if any), and the specific series of user actions (if any). 

Optionally, you can write the URL with the double brackets `{{ }}` syntax to avoid URL encoding. For example, both `https://www.google.com/?q="hello world"` and `https://www.google.com%2F%3Fq%3D%22hello+world%22` are acceptable.

## Customize a context link

When the expected URL output contains a complex pattern that involves a domain name and deep URL parameters, use template variables as values in your context link URL. 

1. Add a link description in the **Label** field such as `See Hello World`.
2. Enter a valid destination link with an arbitrary value in the **URL** field such as `www.google.com/?q="hello world"`.
3. Replace the arbitrary value with a relevant template variable such as `www.google.com/?q="{{source.value}}"`.
4. Click **Save**.

When you click in your dashboard widget, the **See Hello World** link appears as a data action in the dropdown menu.

```